d2bs/kolbot/libs/bots/BattleOrders.js

```diff
@@ -5,6 +5,43 @@
 */
 
 function BattleOrders() {
+	this.AmTardy = function() {
+		let party;
+
+		if (Config.BattleOrders.SkipIfTardy) {
+			party = getParty();
+
+			AreaInfoLoop:
+			while (true) {
+				if (Misc.getPlayerCount() <= 1) {
```

Comment:
Check if it been alone for 10 seconds?
Or, wait until the game is atleast 20 seconds up, something like this?

Reply:
yep, good idea. I'll update